src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Create and manage your accounts
+translate:
+  - jp
+metaDescription: How to add accounts to your New Relic organization. 
+---
+
+import accountsManageSecurity from 'images/accounts_diagram_manage-security.webp'
+
+import accountsMultipleAccountsforLargeDataLimits from 'images/accounts_diagram_multiple-accounts-for-large-data-limits.webp'
+
+import accountsUniqueDataSets from 'images/accounts_diagram_unique-data-sets.webp'
+
+import accountsMultipleAccountEnvironments from 'images/accounts_diagram_multiple-account-environments.webp'
+
+Before you get started with New Relic, it's important to take a moment to think about how you want to use the data you're about to send to New Relic. Do you manage a parent organization with a few different subsidiary companies? Are you a small organization that's growing fast and you need to monitor your application at every stage of the development process?  You can easily customize New Relic to meet your organization's specific needs using accounts. 
+
+With New Relic's account structure you can:
+
+* Maximize data ingest
+* Keep data organized 
+* Monitor multiple environments, like staging and development
+* Manage billing across different areas of your organization
+
+This doc will guide you through setting up your account structure. If you're going to create multiple accounts, it is best to do that _before_ you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc. Feel free to revisit this page if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing)
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account structure. 
```

Comment:
This sentence seems more like one I'd expect in the introduction. Consider just having a sentence like this: Make sure you're familiar with some terms that are key to your decision to create additional accounts.